addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -228,8 +228,10 @@
                 <thead>
                     <tr>
                         <th>Date</th>
-                        <th class="w-75">Name</th>
+                        <th style="width: 30%">Lead</th>
```

Comment:
Is it acceptable to use w-25 ?

We usually try to avoid inline styling in the XML files in favor of bootstrap classes or custom classes.